index.html

```diff
@@ -0,0 +1,146 @@
+            <div class="section card">
+                <div class="icon"><i class="fa fa-3x fa-fire"></i></div>
+                <h2>Premium Materials</h2>
+                <p>
+                    Our trombones use the shiniest brass which is sourced locally. This
+                    will increase the longevity of your purchase.
+                </p>
+            </div>
+
+            <div class="section card">
+                <div class="icon"><i class="fa fa-3x fa-truck"></i></div>
+                <h2>Fast Shipping</h2>
+                <p>
+                    We make sure you recieve your trombone as soon as we have finished
+                    making it. We also provide free returns if you are not satisfied.
+                  </p>
+            </div>
+
+            <div class="section card">
+                <div class="icon"><i class="fa fa-3x fa-truck"></i></div>
+                <h2>Quality Assurance</h2>
+                <p>
+                    For every purchase you make, we will ensure there are no damages or
+            faults and we will check and test the pitch of your instrument.
+                  </p>
+            </div>
+        </section>
+        <!--End of Feature section-->
+
+        <!--How it Works Section-->
+        <section id="howItWorks-link">
+            <iframe
+              id="video"
+              height="315"
+              src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0"
+              frameborder="0"
+              allowfullscreen
+            ></iframe>
+          </section>
+          <!--End of How it Works Section-->
+
+          <!--Pricing Section-->
+          <section id="pricing-link" class="flex mg">
+            <div class="product card">
+              <div class="levels">Tenor Trombone</div>
+              <h2>$600</h2>
+              <ol>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum dolor.</li>
+                <li>Lorem ipsum.</li>
+              </ol>
+              <button class="button">Select</button>
+            </div>
+            <div class="product card">
+              <div class="levels">Bass Trombone</div>
+              <h2>$900</h2>
+              <ol>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum.</li>
+                <li>Lorem ipsum dolor.</li>
+                <li>Lorem ipsum.</li>
+              </ol>
+              <button class="button">Select</button>
+            </div>
+            <div class="product card">
+              <div class="levels">Valve Trombone</div>
+              <h2>$1200</h2>
+              <ol>
+                <li>Plays similar to a Trumpet</li>
+                <li>Great for Jazz Bands</li>
+                <li>Lorem ipsum dolor.</li>
+                <li>Lorem ipsum.</li>
+              </ol>
+              <button class="button">Select</button>
+            </div>
+          </section>
```

Comment:
It will be a good practice to add the above highlighted section in your main section. The main section should always contains all the sections of your code after the navbar and before the footer sections. I recommend you to have a look to the [Document and website structure](https://developer.mozilla.org/enUS/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure) for better understanding the html basic structure.